db-tests/src/test/java/TestTriggers.java

```diff
@@ -152,14 +155,18 @@ void clearMissionModels() throws SQLException {
   }
 
   int insertPlan(int missionModelId) throws SQLException {
+    return insertPlan(missionModelId, "2020-1-1 00:00:00");
+  }
+
+  int insertPlan(int missionModelId, String start_time) throws SQLException {
```

Comment:
Would you mind making all function parameters `final` unless there's a good reason not to? We try to default to `final` as much as possible unless there's a good reason not to. You can also set your editor up to flag this for you if you would like.

Reply:
Done